packages/libs/e2e-test-utils/src/lib/setup-vincent-development-environment.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  delegator,
+  delegatee,
+  appManager,
+  funder,
+  getChainHelpers,
+  ensureUnexpiredCapacityToken,
+  type PkpInfo,
+} from '../index';
+import { type PermissionData } from '@lit-protocol/vincent-contracts-sdk';
+import { type Wallet } from 'ethers';
+
+export interface SetupResult {
+  agentPkpInfo: PkpInfo;
+  wallets: {
+    appDelegatee: Wallet;
+    funder: Wallet;
+    appManager: Wallet;
+    agentWalletOwner: Wallet;
+  };
+  appId: number;
+  appVersion: number;
+}
+
+/**
+ * Helper function to set up a Vincent development environment.
+ * This function handles all the necessary setup steps including:
+ * - Checking and funding all required accounts (funder, app delegatee, app manager)
+ * - Registering or updating your app with abilities and policies
+ * - Creating or using an existing agent PKP
+ * - Setting up permissions for the agent PKP
+ * - Ensuring a valid capacity token exists
+ *
+ * @param permissionData permission data containing abilities and their policies
+ * @returns the setup result including agent PKP info, wallets, app ID, and app version
+ * @example
+ * ```typescript
+ * const permissionData = {
+ *   // EVM Transaction Signer Ability has no policies
+ *   // If yours does, you would add the policies to this object
+ *   [bundledVincentAbility.ipfsCid]: {},
```

Comment:
I think we should remove the mention of ETH tx signer, and then add an example config for setting a policy